bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -214,7 +214,7 @@ def config ():
         parser.add_argument('--neuron.inter_degree', type=str, help='Interpolate algorithm (nearest | linear | bilinear | bicubic | trilinear | area)', default='nearest')
         parser.add_argument('--neuron.name', type=str, help='Trials for this miner go in miner.root / (wallet_cold - wallet_hot) / miner.name ', default='advanced_server')
         parser.add_argument('--neuron.checking', action='store_false', help='To check if server settings are correct',default=True)
-        parser.add_argument('--neuron.no_restart', action='store_true', help='if the model should restart', default=False)
+        parser.add_argument('--neuron.no_restart', type=bool, help='If True, train the neuron from the beginning', default=True)
```

Comment:
type bool doesn't work well, lets change it to store_false

Reply:
but then when user type no_restart, they do not want a restart, if we store false, then no no_restart = restart haha

how about we change it back to restart and just remind Jake not to put this flag everywhere XD